README.md

```diff
@@ -5,38 +5,58 @@
   </picture>
 </p>
 
-[![Join Slack](https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack)](https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q)
-![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
+<p align="center">
+  <a href="https://skypilot.readthedocs.io/en/latest/"> 
+    <img alt="Documentation" src="https://readthedocs.org/projects/skypilot/badge/?version=latest">
+  </a>
+  
+  <a href="https://github.com/skypilot-org/skypilot/releases"> 
+    <img alt="GitHub Release" src="https://img.shields.io/github/release/skypilot-org/skypilot.svg">
+  </a>
+  
+  <a href="https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q"> 
+    <img alt="Join Slack" src="https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack">
+  </a>
+  
+</p>
+
+
+<h3 align="center">
+    Run jobs on any cloud, easily and cost effectively
+</h3>
 
 SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
-Use the clouds **easily** and **cost effectively**, without needing cloud infra expertise.
+SkyPilot abstracts away cloud infra burden:
+- Launch jobs & clusters on any cloud (AWS, Azure, GCP)
+- Find availability across zones/regions/clouds
```

Comment:
Just a thought: I found many people not realizing "availability" as an issue. not sure if the word "availability" is too broad. is "Automatically find available GPUs across zones/regions/clouds" more direct? I don't have a perfect solution.

Reply:
+1 to "availability" may not be a problem people think about. Also +1 that I don't see a better wording.  (The purposed wording is restricted to GPUs, which I think is too narrow.)